### DISCUSSION TITLE
[Feature Requests] Support for MSA password filling.

### DISCUSSION THREAD

**Discussion by quintenvandamme** [florisboard/florisboard#1308] [Feedback]
When you enable MSA to prompt your account info in a app you get a fill in option in gboard. 

Would be really handy if florisboard had this to.

**Comment by quintenvandamme** [comment]
support has been added

**Comment by patrickgold** [comment]
If you mean the Android 11 inline autofill feature, FlorisBoard supports this, however it is a bit buggy when used within a browser.
